crates/bevy_ecs/src/system/combinator.rs

```diff
@@ -285,6 +285,17 @@ where
     }
 }
 
+impl<A, B> Clone for IntoPipeSystem<A, B>
```

Comment:
Can this be `#[derive(Clone)]`, like `IntoAdapterSystem`?  I think `CombinatorSystem` needs an explicit impl because it doesn't require `Func: Clone`, but this impl looks like it could be derived.